SConstruct

```diff
@@ -1,6 +1,56 @@
 #!/usr/bin/env python
 import os
+import sys
+from enum import Enum
 
+# Colors are disabled in non-TTY environments such as pipes. This means
+# that if output is redirected to a file, it won't contain color codes.
+# Colors are always enabled on continuous integration.
+_colorize = bool(sys.stdout.isatty() or os.environ.get("CI"))
+
+
+class ANSI(Enum):
```

Comment:
This new class and helpers double the size of the SConstruct.

I think it should be moved to a separate file. There's also already some ANSI color codes in `tools/godotcpp.py`.

Reply:
> There's also already some ANSI color codes in tools/godotcpp.py.

This is part of the godot-cpp submodule, which we don't have locally when trying to print this error :slightly_smiling_face: 

Either way, I've moved it to a `methods.py` file like in the main Godot repository.